rbd/snapshot.go

```diff
@@ -160,3 +162,24 @@ func (snapshot *Snapshot) Set() error {
 
 	return getError(C.rbd_snap_set(snapshot.image.image, c_snapname))
 }
+
+// GetSnapTimestamp returns the timestamp of a snapshot for an image.
+// For a non-existing snap ID GetSnapTimestamp() hits a Ceph assert.
```

Comment:
I would say "may trigger an assertion and crash in the ceph library" or something along those lines. This clarifies that if they're using a fixed version it may not happen and that it's caused by the ceph library code not something we're doing directly.